virtual-cluster/john.md

```diff
@@ -152,15 +175,61 @@ You can download the iso file with:
 wget https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-9.5.0-amd64-netinst.iso
 ```
 
-Edit /etc/resolv.conf of your inner VM and add the nameserver so that web addresses can be resolved
+When installing the debian image, make sure that 'SSH server' and 'standard system utilities' are included in the installation.
+SSH server is required so that you can set up the VM straight away with Ansible playbook.
+
+During installation, specify VM IP within vmbr0's network e.g. 192.168.16.21/20
+
+During installation, specify nameserver as 169.254.169.254
+
+After installation, edit /etc/ssh/sshd_config and set PermitRootLogin and PasswordAuthentication as 'yes'.
+Then from the VM's node copy ssh key into the VM
+```
+ssh-copy-id 192.168.16.21
+```
+After copying over the ssh key, set PermitRootLogin to prohibit-password and PasswordAuthentication to 'no'.
+
+
+##Ansible Playbook
```

Comment:
The header is not rendered properly. Add a space after `##`. Do the same for the rest.